test/runtests.py

```diff
@@ -16,6 +16,7 @@
 
 from mozillapulse import consumers, publishers
 from mozillapulse.messages.test import TestMessage
+from kombu import Exchange
```

Comment:
Oh sorry, no, you're right, it should be with mozillapulse imports, but, as you say, above them. :)